### ISSUE TITLE
Selection styling.

### ISSUE THREAD

**Issue by jason-neal** [DanielAndreasen/SWEETer-Cat#24]
When selecting points on the plot without a z-axis it is hard to identify the selected values. There is only a slight change in alpha between the selected/nonselected values.

This can be changed by adding [selection/non-selection effects](http://bokeh.pydata.org/en/latest/docs/user_guide/styling.html#selected-and-unselected-glyphs) in the plotting lines.

Basically, you do this by adding keywords with selection_ and nonselection_ prefixes. e.f.
```
fig.circle( x, y, ...
              # set visual properties for selected glyphs
              selection_color="firebrick",
              # set visual properties for non-selected glyphs
              nonselection_fill_alpha=0.2,
)
```

Maybe these settings can be played with at some point for improved selection visuals.

**Comment by DanielAndreasen** [comment]
I agree on this. Some points are almost impossible to see. Even with a z-axis.
